pkg/controller/application/application_controller.go

```diff
@@ -238,6 +247,11 @@ func (c *Controller) syncApplication(key string) error {
 	if app.Annotations == nil {
 		app.Annotations = map[string]string{}
 	}
+	overrideRB, ok := app.Annotations[shipper.RolloutBlocksOverrideAnnotation]
+	if ok {
+		overrideRBs := rolloutBlockOverride.NewOverride(overrideRB)
```

Comment:
Some minor concerns about naming: above we have a var called `overrideRB` and here is another one called overrideRBs. The current state of the code doesn't send an explicit signal about the difference between these 2 vars: i.e. what's their types, why another one is needed etc. One of the options I see here is to omit introduction of the new variable and assign the result of the stringification to the Annotations set right away, like:
```
app.Annotations[shipper.RolloutBlocksOverrideAnnotation] = rolloutBlockOverride.NewOverride(overrideRB).String()
```
Another option is to think of a more explicit type difference underlining. And, of course, the 3rd option is to ignore it :-)